MANIFEST.in

```diff
@@ -1,6 +1,10 @@
 global-exclude .git*
+include CHANGELOG
+include classifiers.txt
 include LICENSE.txt
+include setup.cfg
 include versioneer.py
+include _setup_support.py
```

Comment:
These need to be added here to be included inside sdist tarballs (they are needed to build from sdist)